ares_control/MotorCommand[] data